density/tests/test_latest.py

```diff
@@ -16,3 +16,25 @@ def test_latest_data(self):
         # check number of empty rooms
         self.assertEqual(2, len([r for r in json_resp['data']
                                  if not r['client_count']]))
+
+    def test_annotate_fullness_percentage(self):
+        cur_data = [{
+            "client_count": 5,
+            "dump_time": "Sat, 01 Nov 2014 19:45:00 GMT",
+            "group_id": 147,
+            "group_name": "Architectural and Fine Arts Library 1",
+            "parent_id": 146,
+            "parent_name": "Avery"
+        }]
+        result_percent = 5 / 22.0 * 100
```

Comment:
instead of 22.0, we should use what we get from the capacity_group.json file (which you can access from the global variable you created earlier)